tests/layer_tests/pytorch_tests/test_divmod.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy.testing as npt
+import openvino as ov
+import pytest
+import sys
+import torch
+
+
+# do not test via PytorchLayerTest since PytorchLayerTest triggers own TorchScript tracing
+# this test validates TorchScript patched inside ov.convert_model
```

Comment:
You can use `PytorchLayerTest` with `use_convert_model=True`

Reply:
No, we can't use this path because tests create own TorchScriptDecoder object that performs unpatched tracing.
We should directly call `convert_model` on `torch.nn.Module` then TorchScriptDecoder object is created under the hood with the required patching.